### ISSUE TITLE
video not working

### ISSUE THREAD

**Issue by tgoldenberg** [tokafish/pushertc#2]
Hi, I cloned the repo and inserted my pusher key/ids in pusher.rb and in the #chat data-api-key. No video showing up. The page asks for my name and that's it. Same thing in Heroku on the demo. Any ideas? I would really like to get webRTC working with Pusher for a peer-to-peer video chat. Thanks!

**Comment by tokafish** [comment]
@tgoldenberg The heroku demo is currently working for me. What browser are you using? I didn't test it extensively outside of Chrome, unfortunately.

Also, on the Pusher side - make sure to enable client-sent events, although if you're not seeing any video, I doubt that's the problem.

**Comment by tgoldenberg** [comment]
Hi @wless1, thanks for the quick response. I'm not sure why it wasn't working, but the video is working fine now in my clone version. I had to go to my Pusher dashboard and set client events to on, so that may be way. 

However, for the messaging part of the app, I find that when logged in in two different windows, only one user is recieving messages. I tried this on the demo and found the same thing when I enter the site through an incognito window. Any ideas? Other than that, the app works great!
